package.json

```diff
@@ -10,9 +11,12 @@
   },
   "license": "MIT",
   "dependencies": {
-    "elm": "^0.19.1-3",
-    "elm-format": "^0.8.2",
-    "elm-test": "^0.19.1-revision2",
+    "elm": "^0.19.1-5",
+    "elm-test": "^0.19.1-revision9"
```

Comment:
We've upgraded `elm-explorations/test` in `elm.json` but this needs us to also upgrade `elm-test` NPM dependency:
![Screenshot 2022-11-16 at 16 59 54](https://user-images.githubusercontent.com/149425/202230663-148fb7fa-e63e-4bdd-a882-d319f0ca0c45.png)
```suggestion
    "elm-test": "^0.19.1-revision10"
```

Reply:
Oh whoops, I ran `npm upgrade` and apparently it did not do what I expected it to do. Thank you!